src/main/resources/listLookup.json

```diff
@@ -626,8 +622,7 @@
         "welshFriendlyName": "Rhestr Achosion Dyddiol y Llys Apêl (Adran Troseddol)",
         "shortenedFriendlyName": "Court of Appeal (Criminal Division) Daily Cause List",
         "url": "court-of-appeal-criminal-daily-cause-list",
-        "jurisdictions": ["Crime"],
-        "welshJurisdictions": ["Troseddau"],
+        "jurisdictionTypes": ["Court of Appeal (Civil Division)"],
```

Comment:
This should be 'Court of Appeal (Criminal Division)'

Reply:
changed